examples/tutorials/nb_python/replay_tutorial.py

```diff
@@ -245,14 +247,14 @@ def simulate_with_moving_agent(
     duration=2.0,
     agent_vel=np.array([0.4, 0.0, 0.0]),
     look_rotation_vel=-10.0,
-    get_frames=make_video,
+    get_frames=make_video_during_simulation,
 )
 
 # %% [markdown]
 # ## End the episode. Render the episode observations to a video.
 # %%
 
-if make_video:
+if make_video_during_simulation:
```

Comment:
nit: that's an annoying long variable name. Even shortening to make_video_during_sim would help